src/target_intacct/utils.py

```diff
@@ -11,25 +11,39 @@
 def get_input():
     """Read the input from the pipeline and return a dictionary of the Records."""
     input = io.TextIOWrapper(sys.stdin.buffer, encoding="utf-8")
-    input_value = {}
+    input_value = []
 
     # For each line of input, if it has data content (is a record) add the line to the dictionary
     for row in input:
         try:
-            raw_input = singer.parse_message(row).asdict()
+            raw_input = singer.parse_message(row).asdict()            
         except json.decoder.JSONDecodeError:
             logger.error("Unable to parse:\n{}".format(row))
             raise
-        message_type = raw_input["type"]
-        if message_type == "RECORD" and not any(
+        type = raw_input["type"]
+
+        # Streams of type Record contain the inputted data
+        if type == "RECORD" and not any(
             value == "" or value is None for value in raw_input["record"].values()
```

Comment:
you might get a little better readability for
`not any(value for value in raw_input["record"].values() if value)`